hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/metrics/NamenodeBeanMetrics.java

```diff
@@ -855,7 +855,7 @@ public long getNumberOfSnapshottableDirs() {
 
   @Override
   public String getEnteringMaintenanceNodes() {
-    return "N/A";
+    return null;
```

Comment:
Can we add a test?
I don't think we can reproduce the JSON parsing in the browser side but let's do something closer.

Reply:
Thanks for your review @goiri , I think returning "{}" is more friendly than "null". I submitted new code and added tests.